### PR TITLE
feat(setup): repository monorepo setup

### PR DESCRIPTION
**Pull request by scottnath**
### Summary

Initial setup of monorepo

### Added

- lerna
- placeholder: semantic-release-wildcards
- files and setup for a Node.js repo
- GitHub actions files (don't need to set up travis yet)

## Code review

- [ ] The code must be reviewed by at least one Maintainer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
+yarn
+```
+
+## Contribution Guide
+
+Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull requests to us.
+
+
+[Exported Tests]: https://github.com/IBM/exported-tests
+[Carbon Shared Tests]: /carbon-design-system/carbon-shared-tests
```
Comment (elizabethsjudd) [inline README.md]: This will need a full URL

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+### Pull Request Labels
+
+| Label name | `View Issues in Repository` :mag_right: | Description |
+| --- | --- | --- | --- |
+| `work-in-progress` | [view][search-repo-label-progress] | Pull requests that are in progress, more changes will follow |
+| `needs-review` | [view][search-repo-label-need-review] | Pull requests that need code review and approval from maintainers |
+| `under-review` | [view][search-repo-label-under-review] |Pull requests being reviewed by maintainers |
+| `requires-changes` | [view][search-repo-label-changes] | Pull requests that need to be updated based on review comments and then reviewed again |
+| `needs-testing` | [view][search-repo-label-testing] | Pull requests that need manual testing |
```
Comment (elizabethsjudd) [inline CONTRIBUTING.md:85]: Can you make sure these labels exist too

**Review by elizabethsjudd — CHANGES_REQUESTED** [review]
Requesting changes because the URL needs fixed but otherwise things look good.

File: README.md (hunk 1 above)
Comment (scottnath) [inline README.md]:
```suggestion
[Carbon Shared Tests]: https://github.com/IBM/carbon-design-system/carbon-shared-tests
```

**Comment by scottnath** [comment]
@elizabethsjudd labels updated to reflect CONTRIBUTING doc
https://github.com/IBM/test-automation/labels

ready for re-review please

File: README.md [hunk 3]
```diff
@@ -26,4 +26,4 @@ Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our code of conduc
 
 
 [Exported Tests]: https://github.com/IBM/exported-tests
-[Carbon Shared Tests]: /carbon-design-system/carbon-shared-tests
+[Carbon Shared Tests]: https://github.com/IBM/carbon-design-system/carbon-shared-tests
```
Comment (elizabethsjudd) [inline README.md]: Carbon is not under the IBM umbrella.

```suggestion
[Carbon Shared Tests]: https://github.com/carbon-design-system/carbon-shared-tests
```

**Review by elizabethsjudd — CHANGES_REQUESTED** [review]

**Review by elizabethsjudd — APPROVED** [review]

**Pull request merged**